README.md

```diff
@@ -25,6 +25,28 @@ Library to ease the batching of external requests
 
 # Introduction #
 
+## Summary ##
```

Comment:
@fwbrasil I decided to add this because while the README documentation is freaking incredible (it really is), it gets really complicated and wordy very quickly. I thought it would be nice to provide a really fast rundown on what clump is and provide and powerful example